fail2ban/server/filterpoll.py

```diff
@@ -78,6 +79,16 @@ def _delLogPath(self, path):
 		del self.__file404Cnt[path]
 
 	##
+	# Get a modified log path at once
+	#
+	def getModified(self, modlst):
+		for container in self.getLogPath():
+			filename = container.getFileName()
+			if self.isModified(filename):
+				modlst.append(filename)
+		return modlst
```

Comment:
This is a function with a side-effect -- it modifies passed into it modlst and also returns it... either one or another and better just indeed get a modified one, while keeping original one not modified:  e.g. ```modlst = modlst + [filename]```